smoke/smoke-test-application-NPM-CJS/src/loader-npm-rum-2.ts

```diff
@@ -14,7 +14,8 @@ try {
         cookieAttributes: {
             unique: true
         },
-        useBeacon: false
+        useBeacon: false,
+        enableW3CTraceId: true
```

Comment:
suggestion: we could just enable one of the loader files to have w3c traceId enabled so that we can test x-ray events for header types.